### PR TITLE
Change "Recovery certificate(s)" to "Certificate(s) of recovery"

### PR DESCRIPTION
**Pull request by Ein-Tim**
## Description

This PR changes all occurrences of "Recovery certificate(s)" with "Certificate(s) of recovery", as this is the term used in the app.

## What was changed?

The term "Recovery certificate(s)" was replaced with "Certificate(s) of recovery" in some FAQ & glossary entries.

## Screenshots

I hope it is acceptable that I didn't make a screenshot of every change (-:

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@Ein-Tim 
I don't know why the name was changed in the app. Do you know? Why isn't "Vaccination Certificate" changed to "Certificate of Vaccination" as well?

The EU website https://ec.europa.eu/health/ehealth/covid-19_en uses the term "recovery certificate" and the https://www.digitaler-impfnachweis-app.de/en/ site uses both "recovery certificate" and "certificate of recovery".

I suggest in any case to think carefully about changing the title of https://www.coronawarn.app/en/faq/#glossary_recovery_certificate in the Glossary as it then becomes to strange to be listed under "R" whereas the title would start with "C". You could instead write "Recovery, Certificate of".

The Screenshot sections https://www.coronawarn.app/en/screenshots/#ios_recovery_certificate and https://www.coronawarn.app/en/screenshots/#android_recovery_certificate are also labelled "Recovery Certificate" which is easier to find than "Certificate of Recovery".

**Comment by Ein-Tim** [comment]
@MikeMcC399 

> I don't know why the name was changed in the app. Do you know? Why isn't "Vaccination Certificate" changed to "Certificate of Vaccination" as well?

I also don't know, sorry. I just adapted the website to what is now in the app.

> I suggest in any case to think carefully about changing the title of https://www.coronawarn.app/en/faq/#glossary_recovery_certificate in the Glossary as it then becomes to strange to be listed under "R" whereas the title would start with "C". You could instead write "Recovery, Certificate of".

I agree, or the entry should be moved to the letter "R".

**Comment by MikeMcC399** [comment]
@Ein-Tim 

> > I suggest in any case to think carefully about changing the title of https://www.coronawarn.app/en/faq/#glossary_recovery_certificate in the Glossary as it then becomes to strange to be listed under "R" whereas the title would start with "C". You could instead write "Recovery, Certificate of".
> 
> I agree, or the entry should be moved to the letter "R".

I suggest that you view your change to the Glossary on localhost and you will better understand what I was trying to say.

> > The Screenshot sections https://www.coronawarn.app/en/screenshots/#ios_recovery_certificate and https://www.coronawarn.app/en/screenshots/#android_recovery_certificate are also labelled "Recovery Certificate" which is easier to find than "Certificate of Recovery".
> 
> I can change the titles there as well.

I hope you will not change the titles in the Screenshot section. I think it is better if they are not changed.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

> I suggest that you view your change to the Glossary on localhost and you will better understand what I was trying to say.

Sorry, I reviewed them and understand what you mean and I meant to write "Or the entry should be moved to the letter **C**, not R.

> I hope you will not change the titles in the Screenshot section. I think it is better if they are not changed.

I won't make further changes until it is clarified which term should be used (cc @dsarkar). However, if decided to use "Certificate of Recovery", this term should IHMO be used consistently all over the website.

**Comment by dsarkar** [comment]
@MikeMcC399 @Ein-Tim Thanks for discussing this. In general we try to use the same terminology on the website as used in the CWA. In this case it is pretty much the same terminology, however, let's go ahead with the proposed changes.

- Glossary: 
-> I think @MikeMcC399's proposal is the way to go: "Recovery, Certificate of".
- Screenshot sections https://www.coronawarn.app/en/screenshots/#ios_recovery_certificate and 
 https://www.coronawarn.app/en/screenshots/#android_recovery_certificate 
 -> I suggest, following this discussion, to keep  "Recovery Certificate", just a small inconsistency.

**Comment by Ein-Tim** [comment]
@dsarkar 

Okay, I will change this as you say, thanks for the feedback.

@MikeMcC399 Do you want to follow up why the term "Certificate of Recovery" is used in the app?

**Comment by MikeMcC399** [comment]
@Ein-Tim 
> Do you want to follow up why the term "Certificate of Recovery" is used in the app?

I thought at first that the name "Certificate of Recovery" was new in the latest version of the app 2.15. In fact it has been there since the beginning, as you can see from the Version 2.5 screenshots.

- https://www.coronawarn.app/en/screenshots-archive/2-5.html#android_recovery_certificate
- https://www.coronawarn.app/en/screenshots-archive/2-5.html#ios_recovery_certificate 

The terms "Recovery Certificate" and "Certificate of Recovery" were used interchangeably in the 2.5 blog announcement [Corona-Warn-App version 2.5: Users can integrate recovery certificates and certificates for family members](https://www.coronawarn.app/en/blog/2021-07-12-cwa-version-2-5/)

In the title of the blog the term "recovery certificates" is used.

In the text of the blog the two terms are even used in the same sentence:

"They can also add certificates of recovery, vaccination certificates, as well as test and recovery certificates for other people, such as family members."

In the [Privacy Notice for 2.15](https://www.coronawarn.app/assets/documents/cwa-privacy-notice-en.pdf) the term "Certificate of Recovery" is not used at all, instead the document refers to:

- recovery certificate
- COVID recovery certificate
- digital COVID recovery certificate

---

I suggest therefore to reconsider changing the term used in the FAQ. It is easier for the reader if vaccination certificate, recovery certificate and test certificate are all written in the same way. The Glossary already provides a cross-reference to indicate that "recovery certificate" and "certificate of recovery" are the same thing.

I would prefer to see no change in the FAQs in this respect.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

I understand your points! 

@dsarkar 

Please, could you clarify why the term "Recovery certificate" is not used in the app? 

Thank you!

**Comment by Ein-Tim** [comment]
[`e87d194`](https://github.com/corona-warn-app/cwa-website/pull/2263/commits/e87d194db4ef22f4e67dce6d3c419ef4beab6a89) resolved merge conflicts with `master` branch.

**Comment by MikeMcC399** [comment]
@Ein-Tim / @dsarkar 

I will submit a change request for the app translation to use "recovery certificate" consistently. The term "certificate of recovery" is problematic if the term "certificate of vaccination" and "certificate of test" are not being used.

**Comment by MikeMcC399** [comment]
@Ein-Tim / @dsarkar 

I have submitted https://github.com/corona-warn-app/cwa-documentation/issues/789 to request that the app change the translation to "Recovery Certificate".

Even if this is not done, I would still prefer no change to the FAQs. I believe that changing the FAQ would make it more confusing for readers than leaving it as it is.

**Comment by dsarkar** [comment]
Good morning @Ein-Tim @MikeMcC399, 

Sure, I understand. Thanks for opening the new issue, will be mirrored shortly.

**Comment by MikeMcC399** [comment]
@dsarkar / @Ein-Tim 

Thank you! And on the last day of 2021 thank you for all the interactions and cooperation during this year!
All the best to both of you, to the Open Source Team in the background and all other contributors for a good next year 2022! 🍾 🎇

**Comment by dsarkar** [comment]
@dsarkar / @Ein-Tim 

Thanks a lot for the wishes, which for sure I will forward internally to the team! All the best wishes for 2022 to you and to all the community contributing in a fantastic and constructive way here to this project! Thanks!

**Comment by Ein-Tim** [comment]
Setting to draft until clarified which term should be used.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

Thank you for opening the issue! Have a nice day & a "Guten Rutsch!".

@dsarkar

I can only repeat what has been said by @MikeMcC399, thanks for the great cooperation between the community & you! It's always a pleasure to contribute. Enjoy Friday & "Guten Rutsch!".

**Comment by MikeMcC399** [comment]
@Ein-Tim 

Based on the research in https://github.com/corona-warn-app/cwa-documentation/issues/789#issuecomment-1008339907 the term "Certificate of recovery" is correct, however "recovery certificate" may also be used.

## Suggestions for next steps

1. Rebase this PR on current master
2. Review changes in text to "certificate of recovery" 
3. Make no change (or revert any change in the PR) in places where the term "recovery certificate" is used in a one-line list, such as in https://www.coronawarn.app/en/faq/#eu_dcc_validity "The following text refers to EU digital COVID certificates issued in Germany, i.e. vaccination, test and recovery certificates."
4. Rename the glossary entry "Recovery Certificate" (https://www.coronawarn.app/en/faq/#glossary_recovery_certificate) to "Certificate of Recovery" and move to "C" letter section  
5. Add a new alternate entry "Recovery certificate" with the anchor `recovery_certificate_alt` / `glossary_recovery_certificate_alt`. Link this entry to `#glossary_recovery_certificate`.

## Reference

- [Regulation (EU) 2021/953 of the European Parliament and of the Council of 14 June 2021 on a framework for the issuance, verification and acceptance of interoperable COVID-19 vaccination, test and recovery certificates (EU Digital COVID Certificate) to facilitate free movement during the COVID-19 pandemic (Text with EEA relevance)](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32021R0953)<br>
Regulation (EU) 2021/953 Article 7 defines the "Certificate of recovery".<br>
In the title of the document, and in other lists, the certificate is alternatively referred to as "recovery certificate". See for instance Article 1 - Subject matter:
<br>"This Regulation lays down a framework for the issuance, verification and acceptance of interoperable COVID-19 vaccination, test and recovery certificates (EU Digital COVID Certificate) for the purpose of facilitating the holders’ exercise of their right to free movement during the COVID-19 pandemic."

## Additional information

I opened a new issue on this subject:
- https://github.com/corona-warn-app/cwa-documentation/issues/802

**Comment by Ein-Tim** [comment]
@MikeMcC399 

I'll do as you suggest when someone from the Open-Source Team confirms that they want this.

File: src/data/faq.json [hunk 1]
```diff
@@ -149,8 +149,8 @@
...
+                            "<ul>  <li>digital vaccination certificate</li>  <li>digital certificate of recovery</li>  <li>digital test certificate</li> </ul>",
+                            "Previously, support for digital vaccination certificates was introduced in <a href='../../en/blog/2021-06-10-cwa-version-2-3' target='_blank' rel='noopener'>version 2.3</a>, followed by the addition of support for digital test certificates in <a href='../../en/blog/2021-06-24-cwa-version-2-4' target='_blank' rel='noopener'>version 2.4.</a> The release of <a href='../../en/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>version 2.5</a> added support for digital certificates.",
```
Comment (MikeMcC399) [inline src/data/faq.json:153]: added support for digital certificates.",

should be

added support for digital certificates of recovery.",

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
One of the changes has lost a word. I have marked it up in-line.

Before making any more suggestions, I will wait for the feedback from the Open Source Team that you are also waiting for.

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json:153]: Thanks Mike! I'll fix this with the approval of our plan for the next steps from the Open Source Team. 
Enjoy the evening!

**Comment by Ein-Tim** [comment]
@dsarkar Could I get feedback how to proceed with this, please?

**Comment by Ein-Tim** [comment]
I'll resolve merge conflicts to this PR now. I will do as @MikeMcC399 suggest when @dsarkar confirms that this is OK for the Open Source Team.

**Comment by Ein-Tim** [comment]
Merge conflicts where resolved with [`126dbee`](https://github.com/corona-warn-app/cwa-website/pull/2263/commits/126dbeed7bcf775376df03fc90033e55267621fc).

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Fixed in [`1ab092c`](https://github.com/corona-warn-app/cwa-website/pull/2263/commits/1ab092c90bbb0b31d29ee45b8476ba2e220f8ccb).

**Comment by Ein-Tim** [comment]
Continues here #2491.

**Pull request closed without merging**